demo/api/src/user-permission/config.ts

```diff
@@ -0,0 +1,40 @@
+import { FindUsersArgs, User, UserPermissionConfigInterface, USERPERMISSIONS } from "@comet/cms-api";
+import { staticUsers } from "@src/auth/auth.module";
+
+export const userPermissionConfig: UserPermissionConfigInterface = {
+    getUser: (id: string) => {
+        const index = parseInt(id) - 1;
+        if (staticUsers[index]) return staticUsers[index];
+        throw new Error("User not found");
+    },
+    findUsers: (args: FindUsersArgs) => {
```

Comment:
in a "real" application you will use a mikro-orm repository here, right?

how do you inject it?

Reply:
Would this be a service in a "real" application?

Reply:
Yes: https://github.com/vivid-planet/comet/pull/1223#discussion_r1315493131